packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx

```diff
@@ -104,6 +110,11 @@ export class ChromeService {
     );
   }
 
+  public setup({ analytics }: SetupDeps) {
+    const docTitle = this.docTitle.setup({ document: window.document });
+    registerAnalyticsContextProvider(analytics, docTitle.title$);
+  }
```

Comment:
Had to introduce a `setup` method to the chrome service, given analytic context registration occurs during this stage.